### PR TITLE
NAS-109244 / 21.02 / Add "show_if" to all types / NAS-109244

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/applications/forms/chart-form.component.ts [hunk 1]
```diff
@@ -178,6 +178,11 @@ export class ChartFormComponent {
     if (fieldConfig) {
 
       if (fieldConfig['type']) {
+
+        if (schemaConfig.schema.show_if) {
+          fieldConfig['relation'] = this.createRelations(schemaConfig.schema.show_if, parentName);;
```
Comment (sonicaj) [inline src/app/pages/applications/forms/chart-form.component.ts:183]: Extra `;` ?

**Review by sonicaj — APPROVED** [review]

**Review by sonicaj — CHANGES_REQUESTED** [review]
Actually no, it is not working as intended

File: src/app/pages/applications/forms/chart-form.component.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/applications/forms/chart-form.component.ts:183]: Sure

**Comment by AlexKarpov98** [comment]
It should work, please check again.

**Review by sonicaj — APPROVED** [review]
Yup it was browser cache. Thanks

**Review by dszidi — APPROVED** [review]
LGTM

**Pull request merged**